extensions/observability-devservices/testcontainers/src/main/java/io/quarkus/observability/testcontainers/LgtmContainer.java

```diff
@@ -166,6 +184,31 @@ public Optional<Set<String>> networkAliases() {
         public String otlpProtocol() {
             return ContainerConstants.OTEL_HTTP_PROTOCOL;
         }
+
+        @Override
+        public int scrapingInterval() {
+            return 5;
+        }
+
+        @Override
+        public Optional<Boolean> forceScraping() {
+            return Optional.empty();
+        }
+
+        @Override
+        public String otelMetricExportInterval() {
+            return "10s";
+        }
+
+        @Override
+        public String otelBspScheduleDelay() {
+            return "3s";
```

Comment:
If you really want this to be configurable I would place the values here and in the `LgtmConfig`. They must be set only in one place.

Reply:
OK, moved all to an existing ContainerConstants class.